src/js/components/Dashboard/Overview/index.js

```diff
@@ -17,18 +24,24 @@ export default class extends Component {
   }
 
   render() {
-    const { wallets, coins } = this.props
-    return (
+    const { coins, wallets } = this.props
+    return coins.fetched && wallets.fetched ? ( // make sure app state contains what i need
       <ContentComponent
         header="Overview"
-        subHeader="Manage your assets at a glance"
+        subHeader="Manage your assets at a glance."
       >
-        {!coins.fetched || !wallets.fetched ? (
-          <Loader active />
-        ) : (
-          <HeaderStatistics wallets={wallets.by_name} coins={coins.by_symbol} />
-        )}
+        <link
```

Comment:
it looks like this file already exists within `node_modules/chartist/dist/`
can you just import that?